tools/pinning/current/pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache License 2.0"
 python = "^3.8"
 
 # workaround for: https://github.com/python-poetry/poetry-plugin-export/issues/183
-urllib3 = ">=1.25.4,<1.27"
```

Comment:
Not sure if the work around still applies for this update. Let me know if I should amend this version set up.

Reply:
yeah i think we should still set a maximum `urllib3` version as that poetry-plugin-export issue is unresolved

there's more context at https://github.com/certbot/certbot/issues/9862#issuecomment-2163743729